### PR TITLE
Documentation Fix

### PR DESCRIPTION
**Pull request by JonathanHemnes**
There was a small omission in the findOneAndUpdate() documentation.  Please let me know if you would like me to change any of the language.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -585,11 +585,12 @@ Finds one document matching a `filter` where:
     - `err` - if the query failed, the error reason, otherwise `null`.
     - `result` - if the query succeeded, a document as a class instance.
 
-#### `findOneAndUpdate(filter, [options], callback)`
+#### `findOneAndUpdate(filter, update, [options], callback)`
 
 Finds one document matching a `filter`, updates it and returns it where:
 
  - `filter` - a filter object used to select the document to update.
+ - `update` - The update document.
```
Comment (jedireza) [inline README.md]: Let's use the same working as `findByIdAndUpdate`

`update` - an object containing the fields/values to be updated.

**Comment by jedireza** [comment]
Good find! Thanks for creating a PR. I called out a small change. Once that's fixed I'll merge.

**Comment by jedireza** [comment]
Also, be sure to squash your commits into one.

**Comment by JonathanHemnes** [comment]
Thanks, I'll open up a new PR with the above changes and close this one out!

**Pull request closed without merging**